packages/protocol/contracts/L1/TaikoConfig.sol

```diff
@@ -11,38 +11,44 @@ import { TaikoData } from "../L1/TaikoData.sol";
 library TaikoConfig {
     function getConfig() internal pure returns (TaikoData.Config memory) {
         return TaikoData.Config({
+            // Group 1: general configs
             chainId: 167,
+            relaySignalRoot: false,
+            // Group 2: block level configs
             // Two weeks if avg block time is 10 seconds
-            maxNumProposedBlocks: 120_960,
-            ringBufferSize: 120_960 + 10,
+            blockMaxProposals: 120_960,
+            blockRingBufferSize: 120_960 + 10,
             // Each time one more block is verified, there will be ~20k
             // more gas cost.
-            maxVerificationsPerTx: 10,
+            blockMaxVerificationsPerTx: 10,
             // Set it to 6M, since its the upper limit of the Alpha-2
             // testnet's circuits.
             blockMaxGasLimit: 6_000_000,
+            blockFeeBaseGas: 20_000,
+            // Set it to 79  (+1 TaikoL2.anchor transaction = 80),
+            // and 80 is the upper limit of the Alpha-2 testnet's circuits.
+            blockMaxTransactions: 79,
             // Set it to 120KB, since 128KB is the upper size limit
             // of a geth transaction, so using 120KB for the proposed
             // transactions list calldata, 8K for the remaining tx fields.
-            maxBytesPerTxList: 120_000,
-            proofCooldownPeriod: 30 minutes,
-            systemProofCooldownPeriod: 15 minutes,
-            // Only need 1 real zkp per 10 blocks.
-            // If block number is N, then only when N % 10 == 0, the real ZKP
-            // is needed. For mainnet, this must be 0 or 1.
-            realProofSkipSize: 10,
-            // Set it to 79  (+1 TaikoL2.anchor transaction = 80),
-            // and 80 is the upper limit of the Alpha-2 testnet's circuits.
-            maxTransactionsPerBlock: 79,
+            blockMaxTxListBytes: 120_000,
+            blockTxListExpiry: 0,
+            // Group 3: proof related configs
+            proofRegularCooldown: 30 minutes,
+            proofOracleCooldown: 15 minutes,
+            proofMinWindow: 10 minutes,
+            proofMaxWindow: 90 minutes,
```

Comment:
The `proofMinWindow` will make sure we always allow certain slow proofs to be submitted. But we can tune this parameter value later. Giving it a very small value is equivalent of removing it.

Reply:
Assume A4 testnet will have a CPU perf of 400s, and the GPU might be less than 40s. It means, average proof time is 40s. Then 40*2=80s < 10 min, it will always use 10mins. Thus, in coming A4 testnet, the min window may be too big.

Reply:
Based on the expected CPU performance of 400s for the A4 testnet, and assuming the GPU performance is less than 40s, the average proof time would be around 40s. This indicates that the minimum window for the upcoming A4 testnet may be too large as even 80s (which is double the average proof time) is less than 10 minutes.

Reply:
Based on the performance estimates for the A4 testnet, the CPU is expected to take around 400 seconds, while the GPU may take less than 40 seconds. This indicates that the average proof time will be 40 seconds. If we multiply this by 2, we get 80 seconds, which is less than 10 minutes. Therefore, the minimum window for the upcoming A4 testnet may be too large.

Reply:
Noted, but these values are still very temporary and we expect more circuits to be integrated, so sooner, it's going to be much bigger than 400 seconds.